base/db/R/insert.format.vars.R

```diff
@@ -1,28 +1,67 @@
 #' Insert Format and Format-Variable Records 
-#' @param con Bety connection object
+#'
+#' @param con SQL connection to BETYdb
+#' @param format_notes Character string that describes format.
+#' @param format_name Character string 
+#' @param header Boolean that indicates the presence of a header in the format 
+#' @param skip Integer that indicates the number of lines to skip in the header. Defaults to "" 
+#' @param mimetype_id Integer 
 #' @param formats_variables_df A 'data.frame' consisting of entries that correspond to columns in the formats-variables table
-#' @param formats_df A 'data.frame' consisting of entries that correspond to columns in the formats table
+#' @details It is very important that the formats_variables_df 'data.frame' be structured in a specific format so that the SQL query functions properly. First, all arguments should be passed as vectors so that each entry will correspond with a specific row.
+#' Variables: 
+#' \enumerate{
+#' \item variable_id: vector of integers
+#' \item name: vector of character strings
+#' \item unit: vector of character strings
+#' \item storage_type: vector of character strings
+#' \item column_number: vector of integers
```

Comment:
Similarly, this section should
1. make it clear which variables are required (variable_id) and which are optional (everything else)
2. Explain what actually goes in each column (not just its type) and when it's OK to leave values blank
3. if a column only needs a few values, what should the user put in the empty cells? NULL? NA?

In particular, be aware that `name` is the variable name in the file (e.g. column head) and only needs to be specified if it's different from the BETY variable name. `unit` needs to be udunit2 parsable and only needs to be specified if the units are different from the BETY standard. `column number` only needs to be specified for tabular data that doesn't have a header. `storage_type` serves 2 purposes. First, it indicates columns stored in a format different from expected (e.g. numbers stored as characters). Second, it stores the POSIX codes used to parse any time variables (e.g. a column with a 4-digit year would be `%Y`). This last bit should include a "see also" to `strptime`

Reply:
Clear on all fronts. These changes are in the most recent commit